src/node_i18n.cc

```diff
@@ -508,8 +514,11 @@ static void ToASCII(const FunctionCallbackInfo<Value>& args) {
   CHECK_GE(args.Length(), 1);
   CHECK(args[0]->IsString());
   Utf8Value val(env->isolate(), args[0]);
+  // optional arg
+  bool lenient = args[1].As<Boolean>()->Value();
+
   MaybeStackBuffer<char> buf;
-  int32_t len = ToASCII(&buf, *val, val.length());
+  int32_t len = ToASCII(&buf, *val, val.length(), lenient);
 
   if (len < 0) {
     return env->ThrowError("Cannot convert name to ASCII");
```

Comment:
Is this error part of any non-experimental API? Could we change it to `Cannot encode name to ASCII as Punycode`?